beamer/chain.py

```diff
@@ -459,6 +459,9 @@ def maybe_withdraw(claim: Claim, context: Context) -> None:
         if context.address in winning_addresses:
             _withdraw(claim, context)
 
+    if claim.is_invalidated_l1_resolved and agent_is_challenger:
+        _withdraw(claim, context)
+
```

Comment:
I think we have to include that into the `if / elif condition`. Equal to the `has_l1_resolution` the timestamp doesn't matter if the fill hash is invalidated. So it should be prioritized higher than the claim termination.
In addition to that I think it is possible that we call `_withdraw()` twice the way it is right now.

Reply:
✔️